### PR TITLE
Add new tests for ignoredisk and clearpart commands

### PR DESCRIPTION
**Pull request by poncovka**
* Show tests that will be run.
* Add `autopart `tests to the storage category.
* Add a test for `ignoredisk --only-use`.
* Add `clearpart` tests.

### CODE REVIEW

File: clearpart-2.ks.in [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+umount /sdb
+rmdir /sdb
+%end
+
+%post --nochroot
+SYSROOT=${ANA_INSTALL_PATH:-/mnt/sysimage}
+
+mkdir /sda
+mount /dev/sda1 /sda
+if [[ $? == 0 ]]; then
```
Comment (jkonecny12) [inline clearpart-2.ks.in]: Is there a reason why you don't have a check that it wasn't mounted properly?

File: clearpart-2.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Chris Lumens <clumens@redhat.com>
```
Comment (jkonecny12) [inline clearpart-2.sh]: Fix year and author please.

File: clearpart-3.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Chris Lumens <clumens@redhat.com>
```
Comment (jkonecny12) [inline clearpart-3.sh]: Year and author here too.

File: ignoredisk-1.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Chris Lumens <clumens@redhat.com>
```
Comment (jkonecny12) [inline ignoredisk-1.sh]: The same here.

File: ignoredisk-2.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Chris Lumens <clumens@redhat.com>
```
Comment (jkonecny12) [inline ignoredisk-2.sh]: Also here.

File: clearpart-2.ks.in (hunk 1 above)
Comment (poncovka) [reply clearpart-2.ks.in]: /dev/sda1 was cleared, so I am not sure if it is an error if you cannot mount it anymore.

**Comment by poncovka** [comment]
Fixed.

**Pull request merged**